### PR TITLE
deploy

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

File: workshop/05-deploy.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+_Note: take notice of the org and space you created this app in. It is likely you only have one of each, but it is not uncommon to have more than one of one or the other._
+
+The page that is presented after creating your Node.js app shows your app starting up. We can now switch back to the terminal and our editor.
+
+On this page, you will also see some instructions for using a sample app and getting started. We are going to skip that in favor of the following steps:
+
+## Create our `manifest.yml` file
+
+Go into the root of your Node.js application and create a `manifest.yml` file and add the following information, changing "name" to the name you chose when you created the Cloud Foundry app.
```
Comment (dokun1) [inline workshop/05-deploy.md]: We'll have to check for voice here, if we do want to be consistent.

Comment (joesepi) [reply workshop/05-deploy.md]: Good catch.

File: workshop/05-deploy.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
+# Deploy
+
+LoopBack is simply a Node.js app, so you can deploy it wherever Node.js is supported. In this step though, we will walk through how to deploy it on Bluemix, IBM's Paas (platform as a service). We will use their Cloud Foundry tools to configure and manage the deployment.
```
Comment (dokun1) [inline workshop/05-deploy.md]: "so we can deploy it"

File: workshop/05-deploy.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+![Bluemix - Cloud Foundry Apps](assets/cloud-foundry-apps.png)
+
+**Create the Node.js app**
+
+![Bluemix - Create Node.js app](assets/create-nodejs-app.png)
+
+_Note: take notice of the org and space you created this app in. It is likely you only have one of each, but it is not uncommon to have more than one of one or the other._
+
+The page that is presented after creating your Node.js app shows your app starting up. We can now switch back to the terminal and our editor.
```
Comment (dokun1) [inline workshop/05-deploy.md]: "after creating our Node.js app shows our app starting up"

File: workshop/05-deploy.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+On this page, you will also see some instructions for using a sample app and getting started. We are going to skip that in favor of the following steps:
```
Comment (dokun1) [inline workshop/05-deploy.md]: "on this page, we will also see"

File: workshop/05-deploy.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+## Create our `manifest.yml` file
+
+Let's go into the root of our Node.js application and create a `manifest.yml` file and add the following information, changing "name" to the name you chose when creating the Cloud Foundry app.
```
Comment (dokun1) [inline workshop/05-deploy.md]: "to the name we chose"

File: workshop/05-deploy.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+Now we need to set the region for Bluemix. There are only 3, so we just need to match it up with the region our Org/Space is using in Bluemix. Here are the three options:
+
+|API endpoint                   |Region
+|---                            |---
+|https://api.ng.bluemix.net	|US South
+|https://api.eu-gb.bluemix.net	|United Kingdom
+|https://api.au-syd.bluemix.net	|Sydney
+
+The region that I am using is US South, so you will see that set below (and also in the output above).
```
Comment (dokun1) [inline workshop/05-deploy.md:104]: "that we are using is US south, so we will see that set below"

File: workshop/05-deploy.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+urls: band-aid.mybluemix.net
+last uploaded: Tue Feb 28 22:15:28 UTC 2017
+stack: cflinuxfs2
+buildpack: SDK for Node.js(TM) (ibm-node.js-4.7.2, buildpack-v3.10-20170119-1146)
+
+     state     since                    cpu    memory           disk           details
+#0   running   2017-02-28 05:16:58 PM   0.0%   117.7M of 256M   140.4M of 1G
+```
+
+At this point, our app is in the cloud and you can visit at the urls stated in that output above: `urls: band-aid.mybluemix.net` (_your url may be slightly different)
```
Comment (dokun1) [inline workshop/05-deploy.md]: "and we can visit at the urls"

File: workshop/05-deploy.md (hunk 6 above)
Comment (joesepi) [reply workshop/05-deploy.md:104]: Yeah, with this one, I struggled. But the fact is that I am using US South, but someone else doing this workshop may not be, so I will leave this one as is. Thanks!

**Review by dokun1 — APPROVED** [review]

**Pull request merged**